zips/draft-ecc-zbloc.md

```diff
@@ -0,0 +1,268 @@
+
+A "Decision" is the approval or rejection of a Proposal.
+
+
+# Abstract
+
+In recent polls about the future of Zcash development funding, the community signaled a strong preference for a non-direct funding model. It also decided on a temporary one-year continuation of a development fund, starting at the second halving in November 2024, where ZCG would continue to be funded and the remainder would be temporarily placed in a Deferred Dev Fund Lockbox, while the community determines next steps.
+
+This ZIP outlines a more detailed proposal for non-direct funding, based on voting among a group of Constituencies with a stake in Zcash and its protocol. It has 80% of the block rewards given to miners and 20% to a Zcash Community Fund, in perpetuity. The current balance of the Deferred Dev Fund Lockbox is used to seed the Zcash Community Fund. This fund is initially split into two Funding Pools, with different rules for each: the Large Grant Fund and the Minor Grant Fund. Other specialized pools could be added over time.
+
+This model allows for the community to guide its evolution and possible replacement via Governance Decisions, and so would be activated indefinitely.
+
+
+# Motivation
+
+Zcash governance must represent voices from all over the world, who need digital cash for their security and for their dignity. It must not be dependent on individuals who, despite their best intentions, are flawed and prone to temptations to control and coerce. It must be durable and withstand attempted capture of all forms.
+
+zBloc is a governance model that is designed for decentralization and accountability, to include more voices and ensure results from those who receive Protocol-defined Ecosystem Funding.
+
+It is built to allow new innovative and positive voices to rise up and carry the flag of the Zcash community forward, in recognition of the risk that any one group may become captured or ineffective.
+
+
+# Requirements
+
+* This ZIP covers how to decide on Proposals for potential changes to Zcash Governance and/or disbursement of Protocol-defined Ecosystem Funding.
+* There is a well-defined, publically agreed, process for evaluation and community feedback on both Governance Proposals and Grant Proposals.
+* Funds are held in a multisig resistant to compromise of some of the parties' keys, up to a given threshold.
+* The voting mechanism used for decision-making is similarly resistant to compromise of some of the parties' keys, up to a given threshold.
+* No single party's non-cooperation or loss of keys is able to cause any Protocol-defined Ecosystem Funding to be locked up unrecoverably.
+* 80% of the block rewards are given to miners, and 20% to a Zcash Community Fund seeded from the Deferred Dev Fund Lockbox.
+* The funds from the 20% funding stream will be usable immediately.
```

Comment:
"immediately" is not well-defined; same issue as https://github.com/zcash/zips/pull/989/files#r2003460116